client/account-settings/views/payment-method/index.js

```diff
@@ -4,10 +4,9 @@ import FormRadio from 'components/forms/form-radio';
 import { sprintf } from 'sprintf-js';
 import { translate as __ } from 'lib/mixins/i18n';
 
-// TODO - create PaymentMethod like https://github.com/Automattic/wp-calypso/blob/29984e5677960bdba231577a253cf864d4f17796/client/my-sites/upgrades/checkout/stored-card.jsx
-// or https://github.com/Automattic/wp-calypso/tree/29984e5677960bdba231577a253cf864d4f17796/client/my-sites/upgrades/checkout
-
 const PaymentMethod = ( { value, currentValue, onChange, name, cardType, cardDigits, expiry } ) => {
+	const selected = ( value === currentValue );
+	const containerClasses = selected ? 'payment-method__label selected' : 'payment-method__label';
```

Comment:
You could change this line to:
`const containerClass = classnames( 'payment-method__label', { selected } )`
Kind-of overkill for this, but that's what the `classnames` library is for and we've been using it all over the code.